lib/IRGen/GenDistributed.cpp

```diff
@@ -616,6 +616,8 @@ void DistributedAccessor::emit() {
 
   auto params = IGF.collectParameters();
 
+  GenericContextScope scope(IGM, targetTy->getInvocationGenericSignature());
```

Comment:
The fix™ is that this must be made before the IGM getTypeInfo call, otherwise the generic context is not available (null) and we crash.